types/function-bind/implementation.d.ts

```diff
@@ -0,0 +1,195 @@
+/**
+ * Extracts the type of the 'this' parameter of a function type, or 'unknown' if the function type has no 'this' parameter.
+ */
+// prettier-ignore
```

Comment:
Please don't suppress prettier. I'd like it so that when someone goes back and formats this file, it gets formatted per our formatting rules.